tests/test_utils.py

```diff
@@ -0,0 +1,136 @@
+"""Testing utilities."""
+
+from typing import Callable, Iterable, Optional, Sequence, TypeVar, Union
+
+import numpy as np
+import numpy.typing as npt
+from decorator import decorator
+from scipy.stats.mstats import mquantiles
+from sklearn.utils import check_random_state
+
+
+def draw_bootstrap_samples(
```

Comment:
See `sklearn.utils.resample` for very similar functionality - is there a reason not to use it (or at least wrap it)?